### DISCUSSION TITLE
Add confirm modal when set permission to all/no access

### DISCUSSION THREAD

**Discussion by Dominic-Preap** [directus/directus#19313] [Feature Requests]
### Summary

In Directus, we can do custom permissions, the annoying part for me is that when I tried to click "Use Custom", accidently click "No Access" and everything fields/item permission just gone. 

![image](https://github.com/directus/directus/assets/14802170/f3c36940-2623-47ba-b749-494f65165559)

### Motivation

It would be nice to have confirm dialog simply ask for changing the access, since we are doing custom permission a lot, so misclicking is unavoidable.  

### Detailed Design

It should be simple dialog asking before confirm.

![image](https://github.com/directus/directus/assets/14802170/5430058d-bd6d-4920-852c-7c934a60804d)

### Requirements List

None

### Drawbacks

I'm not sure.

### Alternatives

It's just something that we have to be really extremly careful. Imagine asking colleague to update one field, then he just accidently click "No Access".

### Adoption Strategy

To avoid some users experience for the annoying modal asking everything they change something, we can set env for skipping modal alert.

**Comment by DanielBiegler** [comment]
Sounds reasonable to me, yeah!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
